t/t1035-flux-account-scrub-old-jobs.t

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+test_description='test removing old job records from the flux-accounting database'
```

Comment:
i think a test for the default (no args == 26 weeks) is missing?  Maybe should stick one more test record in there for something > 6 months old.

Reply:
also, probably should cover the path argument too.

Reply:
Good point, thanks. I just pushed up a test that does both - specifying a path to the DB as well as the default for `scrub-old-jobs` (which should remove a simulated job that "finished" over six months ago).